issueshark/backends/jirabackend.py

```diff
@@ -418,16 +409,17 @@ def _parse_issue_links(self, jira_issue_fields, at_name_jira):
         links = []
         for issue_link in getattr(jira_issue_fields, at_name_jira):
             if hasattr(issue_link, 'outwardIssue'):
-                issue_id = self._get_issue_id_by_system_id(issue_link.outwardIssue.key)
+                issue_id = issue_link.outwardIssue.key
```

Comment:
Yes.  I've identified issues with the get_issue_id_by_system_id function in our new design. It would be beneficial for us to discuss and address these problems.